AppCenter/AppCenterTests/MSServiceNotificationDelegateTests.m

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSServiceNotificationDelegate.h"
+#import "MSTestFrameworks.h"
+
+@interface MSServiceNotificationDelegateTests : XCTestCase
+
+@end
+
+@implementation MSServiceNotificationDelegateTests
+
+- (void)testNoRegisteredDelegate {
+  id appCenterMock = OCMPartialMock([MSAppCenter sharedInstance]);
+  // creating delegates
+  id delegate = OCMProtocolMock(@protocol(MSServiceNotificationDelegate));
```

Comment:
If you don't set the delegate then I don't think you need this delegate at all in this test. Can delete this line and the `OCMReject` line 27. Just by not crashing we confirm that not having delegates registered works.